### DISCUSSION TITLE
Participate in W3C Secure the Web Forward workshop?

### DISCUSSION THREAD

**Discussion by tidoust** [jkowalleck/jkowalleck#2] [HELLO]
Hi Jan,

Together with OpenSSF, OWASP, and OpenJS, W3C organizes a [Secure the Web Forward workshop](https://www.w3.org/2023/03/secure-the-web-forward/), partly to discuss the possible impacts of the secure software supply chain approach to web application developments and to web browsers. This includes looking at the integration of software bills of materials (SBOM) as part of the development and build workflow. You maintain a plugin for webpack to create CycloneDX SBOM. That seems directly relevant to that exploration ;)

We're wondering whether you might be interested to detail your perspective in [a short position statement](https://www.w3.org/2023/03/secure-the-web-forward/cfp.html#submit) and join us to discuss possible improvements to the web platform during the workshop.

Some additional context on the workshop:
- The workshop will be held as a virtual event composed of 3 two-hour virtual sessions held on the 26th, 27th and 28th of September 2023.
- Deadline for the submission of position statements is **27 July 2023**.
- Attendance is free for all invited participants and is open to the public, whether or not W3C Members.
- Workshop proceedings will be public.
- This is a *workshop*, not a conference, and any presentations will be short. Our goal is to actively discuss topics, not to watch presentations.
- The workshop will follow W3C's [Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/).

Let me know if you have questions!

**Comment by jkowalleck** [comment]
Thank you for the invitation, @tidoust .
I've just sent my application. I might draft a positional statement after my invitation got confirmed.
